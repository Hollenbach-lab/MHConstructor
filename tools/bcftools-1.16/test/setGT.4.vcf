##fileformat=VCFv4.2
##contig=<ID=1,assembly=b37,length=249250621>
##reference=file:///lustre/scratch105/projects/g1k/ref/main_project/human_g1k_v37.fasta
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	B	A	C
1	1890	.	A	G	.	.	.	GT:DP	1/1:12	0/0:102	0/1:10	1/1:20
1	1991	.	G	T	.	.	.	GT:DP	1/1:8	0/1:100	0/0:11	0/0:11
