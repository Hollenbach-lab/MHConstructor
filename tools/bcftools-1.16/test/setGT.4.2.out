##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=1,assembly=b37,length=249250621>
##reference=file:///lustre/scratch105/projects/g1k/ref/main_project/human_g1k_v37.fasta
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	B	A	C
1	1890	.	A	G	.	.	.	GT:DP	./.:12	0/0:102	./.:10
1	1991	.	G	T	.	.	.	GT:DP	./.:8	./.:100	./.:11
